yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -222,57 +216,37 @@ func parseFleetConfigLine(line string) AssessMigrationDBConfig {
 	}
 }
 
-func getAssessmentExportDirPath(dbConfig AssessMigrationDBConfig) string {
-	switch {
-	case dbConfig.SID != "":
-		return fmt.Sprintf("%s/%s-%s-export-dir", bulkAssessmentDir, dbConfig.SID, dbConfig.Schema)
-	case dbConfig.ServiceName != "":
-		return fmt.Sprintf("%s/%s-%s-export-dir", bulkAssessmentDir, dbConfig.ServiceName, dbConfig.Schema)
-	case dbConfig.TnsAlias != "":
-		return fmt.Sprintf("%s/%s-%s-export-dir", bulkAssessmentDir, dbConfig.TnsAlias, dbConfig.Schema)
-	default:
-		panic(fmt.Sprintf("unable to get export dir for assessment of schema-%s", dbConfig.Schema))
-	}
-}
-
-func getDatabaseIdentifier(dbConfig AssessMigrationDBConfig) string {
-	switch {
-	case dbConfig.SID != "":
-		return dbConfig.SID
-	case dbConfig.ServiceName != "":
-		return dbConfig.ServiceName
-	case dbConfig.TnsAlias != "":
-		return dbConfig.TnsAlias
-	default:
-		return ""
-	}
-}
-
 //go:embed templates/bulkAssessmentReport.template
 var bulkAssessmentHtmlTmpl string
 
+const REPORT_PATH_NOTE = `To apply recommendations from the bulk assessment, you can use the generated export-dir containing the assessment report (recommended approach), 
```

Comment:
To automatically apply the colocated/sharding recommendations to the exported schema, continue the migration steps(export-schema, import-schema, ..) using the auto-generated export-dirs. 

In case a different export-dir has to be used, you can use the flag --assessment-report-path in export-schema in order to apply the colocated/sharding recommendations

Reply:
how about a bit shorter one:
```
To automatically apply the recommendations, continue the migration steps(export-schema, import-schema, ..) using the auto-generated export-dirs.
If using a different export-dir, specify report path with `--assessment-report-path` flag  to apply the recommendations.
```

Reply:
Sounds good. Just mention the export-schema command. 
"specify report path in export-schema with ... "